components/radioButton/radioButton.js

```diff
@@ -0,0 +1,94 @@
+// Imports
+import React, { PropTypes } from 'react';
+
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+function RadioButton({
+  checked,
+  children,
+  dataAttrs = false,
+  disabled,
+  id,
+  mods = [],
+  name,
+  onChange }) {
+  const restProps = getDataAttributes(dataAttrs);
+
+  if (disabled) {
+    mods.push('disabled_true');
+  }
+
+  const className = getClassNamesWithMods('ui-radio', mods);
+
+  return (
+    <div className={className} {...restProps}>
+      <input
+        checked={checked}
+        disabled={disabled}
+        id={id}
+        name={name}
+        onChange={onChange}
+        type="radio"
+      />
+      <label aria-checked={checked} htmlFor={id} role="radio">
```

Comment:
That is exactly what i mean :) you have label with for='radio' attribute already, aria-checked={checked} role="radio" is not required, so accessibility devices understand this is a label for radio input